src/db.ts

```diff
@@ -24,10 +27,23 @@ export const client = <
 		throw new Error('Missing required environment variable: DB_CONNECTION_STRING');
 	}
 
+	const doLog = c.env.DB_ENABLE_LOGGING === 'true' ? true : false;
+
 	if (c.env.HYPERDRIVE?.connectionString) {
 		dbClient = drizzlePostgres(c.env.HYPERDRIVE.connectionString);
 	} else {
-		dbClient = drizzleNeon(c.env.DB_CONNECTION_STRING);
+		if (c.env.DB_SCHEMA) {
```

Comment:
With this we require the user to set two environment variables when developing, DB_SCHEMA and DB_CONNECTION_STRING?

Is it possible(/how hacky is it?) to keep the `DB_CONNECTION_STRING` as is (with the search_path param), and then parse the connection string here to extract that param and add the appropriate `pool.on('connect', ...`?